### ISSUE TITLE
Ugly faq styles in kudos

### ISSUE THREAD

**Issue by octavioamu** [gitcoinco/web#2844]
**Describe the bug**
[comment]: # (A clear and concise description of what the bug is.)
The current styles of the faq section in kudos is ugly we need to make a revamp. 

**To Reproduce**
[comment]: # (Steps to reproduce the behavior)
Go to https://gitcoin.co/kudos/ and scroll down. 

**Expected behavior**
[comment]: # (A clear and concise description of what you expected to happen.)
Nice / clean styles

**Screenshots**
[comment]: # (If applicable, add screenshots to help explain your problem.)

**Desktop (please complete the following information):**

- OS: xxxx
- Browser xxxx
- Browser Version xxxx

**Smartphone (please complete the following information):**

- Device: xxxx
- OS: xxxx
- Browser [xxxx
- Version xxxx

**Additional context**
**Smartphone (Add any other context about the problem here.)

**Comment by Anish-Agnihotri** [comment]
Here is a quick sample (with PR) that I just pushed out. I wasn't exactly sure what you meant by 'clean styles' so I just followed typical Gitcoin color schemes and styling guides. 

Here is an image of it: ![cleaner](https://user-images.githubusercontent.com/4398624/48594454-9e5d3780-e91e-11e8-9a3c-ef393146971a.JPG)

Of course, if you're looking for a complete overhaul of the FAQ section though, or have any specific ideas in mind, let me know, and I'd be glad to push some new styling out in a few minutes.

**Comment by thelostone-mc** [comment]
@willsputra this look alright to you ? ^_^

**Comment by willsputra** [comment]
@Anish-Agnihotri @thelostone-mc Looks good to me! I'd just change the colors since this one's too... earthy lol. Could we try #F2F6F9 for the background and #0D0764 for the dropdown button?

Also, are we going to have hover states for the boxes?

Thanks for helping with this Anish! 🙂

**Comment by Anish-Agnihotri** [comment]
That sounds like a good idea to me! I'll replace the colors, add hover states, and maybe look into a bit of drop shadow/something to divide the response and the header of the faq-items. 

I'll make another commit within the next few hours!

**Comment by octavioamu** [comment]
Thanks @Anish-Agnihotri for helping with this!

**Comment by Anish-Agnihotri** [comment]
Just pushing out the PR with the new changes. Apologies for the delay, took a bit longer than expected since I'm away from my usual development environment. 

I've changed the `background-color` and the color of the dropdown button. I also added a `#fefefe` hover with a `ease-in-out` animation, but this might be a bit light (I'm design-blind, haha). I tried drop-shadows, although decided to go against them, since it over-complicated the appearance of the accordions. 

Here are some screenshots:

![theme1](https://user-images.githubusercontent.com/4398624/48664269-0704fb00-ea6a-11e8-9ae4-2f9fbd87d280.png)
![hover](https://user-images.githubusercontent.com/4398624/48664270-08362800-ea6a-11e8-832d-48ac074a75fe.png)


As always, let me know ideas/concerns/questions and I'll knock those out and push out another PR :)

Always glad to help out!

Anish
